pkg/volume/vsphere_volume/vsphere_volume.go

```diff
@@ -280,7 +281,9 @@ func (v *vsphereVolumeUnmounter) TearDownAt(dir string) error {
 		return fmt.Errorf("directory %s is not mounted", dir)
 	}
 
-	v.volPath = path.Base(refs[0])
+	//replacing the \040 encoding with space
```

Comment:
@abithap 

It's not clear to me where the space is being introduced (I read the comment in the issue you opened for this as well).  Could you elaborate?